### PR TITLE
Web console: Fix missing include future flag for byPeriod rules

### PR DESCRIPTION
**Pull request by renevan10**
When adding a by period rule, the user can now decide whether to include the future through a flag. By default it is set to true.

image

Fixes [#8855](https://github.com/apache/incubator-druid/issues/8855)

### CODE REVIEW

File: web-console/src/components/rule-editor/rule-editor.scss [hunk 1]
```diff
@@ -30,4 +30,11 @@
       flex: 1;
     }
   }
+
+  .byPeriod {
```
Comment (vogievetsky) [inline web-console/src/components/rule-editor/rule-editor.scss]: please use `kebab-case` for class names

File: web-console/src/utils/load-rule.ts [hunk 2]
```diff
@@ -41,7 +42,13 @@ export class RuleUtil {
     return (
       rule.type +
       (rule.period ? `(${rule.period})` : '') +
-      (rule.interval ? `(${rule.interval})` : '')
+      (rule.interval ? `(${rule.interval})` : '') +
+      (rule.includeFuture !== false &&
+      (rule.type === 'loadByPeriod' ||
+        rule.type === 'dropByPeriod' ||
+        rule.type === 'broadcastByPeriod')
```
Comment (vogievetsky) [inline web-console/src/utils/load-rule.ts]: this logic should be extracted into a utility function

File: web-console/src/views/datasource-view/datasource-view.tsx [hunk 3]
```diff
@@ -642,11 +642,6 @@ GROUP BY 1`;
       ];
     } else {
       return goToActions.concat([
-        {
-          icon: IconNames.GANTT_CHART,
-          title: 'Go to tasks',
-          onAction: () => goToTask(datasource),
-        },
```
Comment (vogievetsky) [inline web-console/src/views/datasource-view/datasource-view.tsx]: why did you remove this?

File: web-console/src/components/rule-editor/rule-editor.scss (hunk 1 above)
Comment (renevan10) [reply web-console/src/components/rule-editor/rule-editor.scss]: I've made the changes

File: web-console/src/utils/load-rule.ts (hunk 2 above)
Comment (renevan10) [reply web-console/src/utils/load-rule.ts]: I've made the changes

File: web-console/src/views/datasource-view/datasource-view.tsx (hunk 3 above)
Comment (renevan10) [reply web-console/src/views/datasource-view/datasource-view.tsx]: I noticed this was duplicated, if you look carefully there will be two "Go to tasks" under each datasource(unless they are disabled).

Comment (vogievetsky) [reply web-console/src/views/datasource-view/datasource-view.tsx]: oh. Great catch!

**Comment by vogievetsky** [comment]
LGTM after CI 👍
